echo360/downloader.py

```diff
@@ -250,9 +250,9 @@ def _initialize(self, echo_course):
 
     def _get_filename(self, course, date, title):
         if course:
-            filename = "{} - {} - {}".format(course, date, title)
+            filename = "{} - {} - {}".format(course, date, title[:150])
```

Comment:
include a comment to explain what this is doing (e.g. avoid filename too long exception)